src/pages/iou/ReceiptSelector/index.native.js

```diff
@@ -75,8 +75,10 @@ function getImagePickerOptions(type) {
 }
 
 function ReceiptSelector(props) {
-    const devices = useCameraDevices('wide-angle-camera');
-    const device = devices.back;
+    const wideAngleCamera = useCameraDevices('wide-angle-camera');
+    const ultraWideAngleCamera = useCameraDevices('ultra-wide-angle-camera');
+    const telephotoCamera = useCameraDevices('telephoto-camera');
+    const device = wideAngleCamera.back || ultraWideAngleCamera.back || telephotoCamera.back;
```

Comment:
This sounds good to me if it works on the Xiaomi A3.

```js
const wideAngleCamera = useCameraDevices('wide-angle-camera');
const fallbackCamera = useCameraDevices();
const device = wideAngleCamera.back || fallbackCamera.back;
```

Reply:
I'll give it a try, I thought I HAD to pass an argument to that function

Reply:
Just a note, the others you are suggesting are "LogicalCameraDeviceType", which I thought was unnecessary because I imagine we should find a physical device if logicals are available... but hard to be sure 🤷